test/integration/testutils/transforms.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2022 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutils
+
+import (
+	"github.com/tidwall/gjson"
+)
+
+// getResultFieldStrSlice parses a field of a results list into a string slice
```

Comment:
I think it would make sense to add this function to `blueprint-test/pkg/utils`: https://github.com/GoogleCloudPlatform/cloud-foundation-toolkit/blob/master/infra/blueprint-test/pkg/utils/transforms.go

@Samir-Cit let me know if I can assist

Reply:
We are planning to move this function (and others we have on foundation) to this repo you mention.
We just didn't do yet to not block the PRs here.
But ASAP we will move all utils to toolkit.

Reply:
@Samir-Cit - Great, feel free to ping me on the PR for review.